src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -209,94 +377,216 @@ static void OnFlyoutBehaviorChanged(BindableObject bindable, object oldValue, ob
+		/// <param name="value">The disabled color for the tab bar.</param>
 		public static void SetDisabledColor(BindableObject obj, Color value) => obj.SetValue(DisabledColorProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetForegroundColor']/Docs/*" />
+		/// <summary>
+		/// Gets the foreground color for the tab bar. 
+		/// </summary>
+		/// <param name="obj">The object to which the foreground color is set.</param>
+		/// <returns>The foreground color for the tab bar.</returns>
 		public static Color GetForegroundColor(BindableObject obj) => (Color)obj.GetValue(ForegroundColorProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetForegroundColor']/Docs/*" />
+
+		/// <summary>
+		/// Defines the foreground color for the tab bar. 
+		/// If the property is unset, the ForegroundColor property value is used.
+		/// </summary>
+		/// <param name="obj">The object to which the foreground color is set.</param>
+		/// <param name="value">The foreground color for the tab bar.</param>
 		public static void SetForegroundColor(BindableObject obj, Color value) => obj.SetValue(ForegroundColorProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetTabBarBackgroundColor']/Docs/*" />
+		/// <summary>
+		/// Gets the background color for the tab bar.
+		/// </summary>
+		/// <param name="obj">The object to which the color is set.</param>
+		/// <returns>The background color for the tab bar.</returns>
 		public static Color GetTabBarBackgroundColor(BindableObject obj) => (Color)obj.GetValue(TabBarBackgroundColorProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetTabBarBackgroundColor']/Docs/*" />
+
+		/// <summary>
+		/// Sets the background color for the tab bar. 
+		/// If the property is unset, the BackgroundColor property value is used.
+		/// </summary>
+		/// <param name="obj">The object to which the color is set.</param>
+		/// <param name="value">The background color for the tab bar.</param>
 		public static void SetTabBarBackgroundColor(BindableObject obj, Color value) => obj.SetValue(TabBarBackgroundColorProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetTabBarDisabledColor']/Docs/*" />
+		/// <summary>
+		/// Gets the background color for the tab bar. 
+		/// </summary>
+		/// <param name="obj">The object to which the color is set.</param>
+		/// <returns>The background color for the tab bar.</returns>
 		public static Color GetTabBarDisabledColor(BindableObject obj) => (Color)obj.GetValue(TabBarDisabledColorProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetTabBarDisabledColor']/Docs/*" />
+
+		/// <summary>
+		/// Sets the disabled color for the tab bar. 
```

Comment:
I think it might be better to expand this a bit to be clearer. 
```suggestion
		/// Sets the color of the tab bar when its disabled. 
```